utils/scripts/find_duplicate_read_sets/find_duplicate_read_sets.py

```diff
@@ -0,0 +1,118 @@
+import boto3
+import pandas as pd
```

Comment:
pandas needs to be called out as a prereq

Reply:
done